### ISSUE TITLE
Al ver fotos sale de la app

### ISSUE THREAD

**Issue by Pablocalvo321** [onelife-social/support#6]
En el teléfono de Talia cuando está vídeo fotos le saca de la App sin decir nada

**Comment by juanan-onelife** [comment]
https://github.com/flutter/flutter/issues/144565

**Comment by jlerma-onelife** [comment]
Se subió la semana pasada la v2.7.7 para que probase con la nueva versión de Flutter. Lo ha probado y parece que ahora se le sale menos, pero le sigue tirando en ocasiones. Ahora parece que tiene este otro error:

https://console.firebase.google.com/project/onelife-social/crashlytics/app/ios:social.onelife/issues/8421aca7ecc9b84b6f8a830aecc50e1f?time=last-seven-days&types=crash&sessionEventKey=70bdaddbe5e64769b81e998831910406_1936472180065426434&sessionTab=data&userInfoQuery=oClhpHe3tWZdIj6sksyltPIbFCC3

Mira a ver si podemos arreglar ese que es el acceso a una imagen que no existe.

**Comment by jlerma-onelife** [comment]
Sí, no hay problema en hacer una versión específica para ella si eso nos ayuda a solventar el error.
